op-supervisor/supervisor/backend/cross/safe_frontier_test.go

```diff
@@ -11,6 +11,55 @@ import (
 	"github.com/ethereum-optimism/optimism/op-supervisor/supervisor/types"
 )
 
+func TestSufficientDataChecks(t *testing.T) {
+	t.Run("empty hazards", func(t *testing.T) {
+		sfcd := &mockSafeFrontierCheckDeps{}
+		l1Source := eth.BlockID{}
+		hazards := map[eth.ChainID]types.BlockSeal{}
+		err := SufficientDataChecks(sfcd, l1Source, NewHazardSetFromEntries(hazards))
+		require.NoError(t, err)
+	})
+	t.Run("latest ahead of scope", func(t *testing.T) {
+		sfcd := &mockSafeFrontierCheckDeps{}
+		l1Source := eth.BlockID{Number: 1}
+		hazards := map[eth.ChainID]types.BlockSeal{}
+		sfcd.crossSafeFn = func(chain eth.ChainID) (latest types.DerivedBlockSealPair, err error) {
+			return types.DerivedBlockSealPair{
+				Source:  types.BlockSeal{Number: 1},
```

Comment:
```suggestion
				Source:  types.BlockSeal{Number: 2},
```

I think it's otherwise the same as the below case.

"latest ahead of scope" as sub-test implies the latest cross-safe value is not of the same scope.

Reply:
good catch, thanks